### DISCUSSION TITLE
how to use `getStorageAt` and ethers other method

### DISCUSSION THREAD

**Discussion by web3dopamine** [wevm/wagmi#2011] [Q&A] [answered]
is it possible to use methods from ethers such as `getStorageAt`?

If yes how?

https://docs.ethers.org/v5/api/providers/provider/#Provider-getStorageAt

**Comment by tmm** [comment, marked as answer]
There isn't a React Hook for `getStorageAt` right now.

You can either use raw ethers or use [viem](https://viem.sh/docs/contract/getStorageAt.html) for this (wagmi will be switching to viem from ethers for `wagmi@1`).
